homeassistant/components/roborock/const.py

```diff
@@ -54,3 +56,10 @@
 MAP_FILENAME_SUFFIX = ".png"
 SET_VACUUM_GOTO_POSITION_SERVICE_NAME = "set_vacuum_goto_position"
 GET_VACUUM_CURRENT_POSITION_SERVICE_NAME = "get_vacuum_current_position"
+
+
+A01_UPDATE_INTERVAL = timedelta(seconds=30)
+V1_CLOUD_IN_CLEANING_INTERVAL = timedelta(seconds=20)
+V1_CLOUD_NOT_CLEANING_INTERVAL = timedelta(seconds=30)
+V1_LOCAL_IN_CLEANING_INTERVAL = timedelta(seconds=10)
+V1_LOCAL_NOT_CLEANING_INTERVAL = timedelta(seconds=20)
```

Comment:
Local not cleaning lets leave at 30 seconds like it was before too?  How about local when cleaning becomes 15 seconds for now.